common/source/docs/common-flywoo-f745.rst

```diff
@@ -70,15 +91,30 @@ UART3 supports RX and TX DMA. UART1, UART2, UART4, and UART6 supports TX DMA. UA
 RC Input
 ========
 
-RC input is configured by default via the USART3 RX input. It supports all RC protocols except PPM. FPort and full duplex protocols will use both RX6 and TX6. The 4.5V output pin is powered when the USB is connected to allow setup of receiver without connecting to the main battery.
+RC input is configured on the RX3 (UART3_RX) pin. It supports all RC protocols except PPM. See :ref:`common-rc-systems` for details for a specific RC system. :ref:`SERIAL3_PROTOCOL<SERIAL3_PROTOCOL>` is set to "23", by default, to enable this.
+
+- SBUS/DSM/SRXL connects to the RX3 pin but SBUS requires that the :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` be set to "3".
 
-.. note:: If the receiver is SBUS, then :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` must be set to "1" to provide inversion for detection. If the receiver is FPort, then the receiver must be tied to the USART3 TX pin and :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` = 7 (invert TX/RX, half duplex) and :ref:`RSSI_TYPE<RSSI_TYPE>` =3.
+- FPort requires connection to TX3 and :ref:`SERIAL3_OPTIONS<SERIAL3_OPTIONS>` be set to "7".
```

Comment:
Ditto here 7 means invert rx and tx and use half-duplex

Reply:
just the values Tridge has stated to use for FPort...its like that everywhere...maybe the the rx does not inversion....depends on how things connect internally when half duplex is used...it is the inversion before or after the internal path mux to the fifos